cmd/tailscale/cli/set.go

```diff
@@ -248,6 +251,26 @@ func runSet(ctx context.Context, args []string) (retErr error) {
 		maskedPrefs.Prefs.RelayServerPort = ptr.To(int(uport))
 	}
 
+	if setArgs.relayServerStaticEndpoints != "" {
+		endpointsMap := map[netip.AddrPort]bool{}
```

Comment:
We have [`sets.Set`](https://godoc.org/tailscale.com/util/set) if you like (with a `Slice` method to save you one of the loops).